package.json

```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "axios": "^0.20.0",
     "body-parser": "^1.19.0",
+    "cassandra-driver": "^4.6.0",
```

Comment:
Once you think you're ready for deployment, you can start to think about cleaning up your dependencies. Anything Cassandra or MySQL related can probably go if you aren't using it, for example. Looking down, enzyme should probably be a dev dependency since it's for testing, although I know you didn't put it there.